modules/core/04-channel/v2/types/msgs.go

```diff
@@ -108,15 +108,13 @@ func (msg *MsgAcknowledgement) ValidateBasic() error {
 		return errorsmod.Wrap(commitmenttypesv1.ErrInvalidProof, "cannot submit an empty acknowledgement proof")
 	}
 
-	if err := msg.Acknowledgement.Validate(); err != nil {
-		return err
-	}
-
 	_, err := sdk.AccAddressFromBech32(msg.Signer)
 	if err != nil {
 		return errorsmod.Wrapf(ibcerrors.ErrInvalidAddress, "string could not be parsed as address: %v", err)
 	}
 
+	// NOTE: Acknowledgement is allowed to be empty if the receive failed
```

Comment:
Note: This annoying point that I do allow empty acks being sent in by relayer as a valid message now for error acks

Reply:
Why do you allow this? Isn't it emitted in the event? This doesn't make much sense to me since you'd need the proof for it anyway